lib/screens/login_screen.dart

```diff
@@ -19,96 +20,107 @@ class _LoginScreenState extends State<LoginScreen> {
     queryData = MediaQuery.of(context);
     return Scaffold(
       appBar: AppBar(),
-      body: SafeArea(
-        child: Center(
-          child: Column(
-            mainAxisAlignment: MainAxisAlignment.center,
-            children: [
-              Padding(
-                padding: const EdgeInsets.all(10.0),
-                child: Row(
-                  mainAxisAlignment: MainAxisAlignment.center,
-                  crossAxisAlignment: CrossAxisAlignment.baseline,
-                  textBaseline: TextBaseline.alphabetic,
-                  children: [
-                    Hero(
-                      tag: 'logo',
-                      child: Container(
-                        child: Padding(
-                          padding: const EdgeInsets.only(bottom: 60),
-                          child: Image(
-                            image: kLogo,
+      body: Form(
+        key: _key,
+        child: SafeArea(
+          child: Center(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Padding(
+                  padding: const EdgeInsets.all(10.0),
+                  child: Row(
+                    mainAxisAlignment: MainAxisAlignment.center,
+                    crossAxisAlignment: CrossAxisAlignment.baseline,
+                    textBaseline: TextBaseline.alphabetic,
+                    children: [
+                      Hero(
+                        tag: 'logo',
+                        child: Container(
+                          child: Padding(
+                            padding: const EdgeInsets.only(bottom: 60),
+                            child: Image(
+                              image: kLogo,
+                            ),
                           ),
+                          height: 100,
                         ),
-                        height: 100,
                       ),
-                    ),
-                    Text(
-                      'Login',
-                      style: TextStyle(fontSize: 60.0, color: Colors.black87),
-                    ),
-                  ],
+                      Text(
+                        'Login',
+                        style: TextStyle(fontSize: 60.0, color: Colors.black87),
+                      ),
+                    ],
+                  ),
                 ),
-              ),
-              Container(
-                child: TextField(
-                  autofocus: false,
-                  decoration: kTextFieldDecor,
-                  maxLines: 1,
-                  onChanged: (value) {
-                    email = value;
-                  },
+                Container(
+                  child: TextFormField(
+                    validator: (input) => input.isValidEmail() ? null : "Check your email",
+                    autofocus: false,
+                    decoration: kTextFieldDecor,
```

Comment:
- Modify `"Check your email"` to `"Enter a valid email"`.
- The validator just returns a string. Kindly reflect the message in the UI as well.